src/include/daos_srv/vea.h

```diff
@@ -330,4 +331,12 @@ void *vea_metrics_alloc(const char *path, int tgt_id);
  */
 int vea_metrics_count(void);
 
+/**
+ * Set the VEA metric defining the NVME usage.
+ *
+ * \param metrics   [IN|OUT]	Metrics to updated
+ * \param nvme_used [IN]	Metric to be set
+ */
+void vea_metrics_nvme_used_set(void* metrics, struct d_tm_node_t* nvme_used);
```

Comment:
(style)  "foo* bar" should be "foo *bar"
(style)  "foo* bar" should be "foo *bar"